src/algebra/invertible.lean

```diff
@@ -161,14 +161,15 @@ inv_of_eq_right_inv (mul_one _)
 def invertible_neg [ring α] (a : α) [invertible a] : invertible (-a) :=
```

Comment:
Shouldn't this one generalize too?

Reply:
although the linter didn't catch it! should've checked